정봉석/vite-project/src/App.jsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import "./App.css";
+import Counter from "../Counter/CounterPage";
+
+function App() {
+  return (
+    <>
+      <div>
+        <div>
+          <Counter />
+        </div>
+      </div>
```

Comment:
div 태그를 필요 이상으로 작성하는 것은 불필요한 DOM 트리를 형성해서 좋은 방법은 아니에요!
추가적인 컴포넌트를 작성하실 게 아니라면 wrapper는 하나정도면 충분 할 것 같습니다!
```suggestion
      <div>
          <Counter />
      </div>
```